src/cloud/lib/props.ts

```diff
@@ -195,3 +206,7 @@ export function getDefaultColumnSuggestionsPerType(): {
     { type: 'date', name: 'Start Date' },
   ]
 }
+
+export function cleanupDateProp(date: Date) {
```

Comment:
`cloud/lib/date` should have everything you need. For this case, `getISODateFromLocalTime` should be the right one.